action.yml

```diff
@@ -21,6 +21,15 @@ inputs:
   sonar-api-url:
     description: Base URL of the SonarCloud API.
     default: https://sonarcloud.io/api
+  defectdojo-token:
+    description: Token for authenticating requests to DefectDojo.
+    required: false
+  defectdojo-api-url:
+    description: Base URL of the DefectDojo API.
+    required: false
```

Comment:
I think this should be required, in https://github.com/pixee/upload-tool-results-action/pull/9/files#diff-21563dca87d9bc102a8ef9005cd9938c01e74be0aa96511fd5ee092b794f8265R45 you define throw

Reply:
I am unable to find any documentation for a default URL. Unlike Sonar, I think DefectDojo is generally self-hosted so it's not like there is one canonical instance that we could refer to here.